controlplane/src/core/services/Keycloak.ts

```diff
@@ -370,4 +425,73 @@ export default class Keycloak {
 
     return viewerChildGroup;
   }
+
+  public async removeUserFromOrganization({
```

Comment:
Please describe a bit the internals, why we need to handle each role differently.

Reply:
Improved

Reply:
when the user is removed from the org, we have to remove his roles from keycloak

Reply:
thanks